packages/signature-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Changed
+- **BREAKING**: Removed `keyringController` property from constructor option ([#1593](https://github.com/MetaMask/core/pull/1593))
```

Comment:
Nit: this could've been mark as changed since it doesn't going to break anything

Reply:
It is not going to raise an error on the removed parameter, but Typescript will complain about the missing messenger actions that replaced the constructor parameter

Reply:
Good point, make sense 👍